### ISSUE TITLE
Cypress 2.x cannot test Angular 5 live-served apps.

### ISSUE THREAD

**Issue by BenAychh** [cypress-io/cypress#1334]
- Operating System: High Sierra
- Cypress Version: 2.0.1
- Browser Version: Chrome 64.0.3282.167

### Is this a Feature or Bug?

Bug

### Current behavior:

When I start my angular app with `ng serve` I am unable to run the cypress tests against that. This was previously working in 1.4.2. Now it complains that my app has an error but there are no errors in the console

### Desired behavior:

Test against localhost:4200

### How to reproduce:

Please check this repo https://github.com/BenAychh/angular-cypress-2.
Run `ng serve` to get the app live-reload-server up and navigate to http://localhost:4200. Check for errors.
Now run the cypress test with the above server still up and try to test.

#### Test code:

```js
describe('Test', () => {
  it('has the correct title', () => {
    cy.visit('/');
    cy.title().should('equal', 'CypressTest');
  });
});

```

### Additional Info (images, stack traces, etc)

I've been using `npx cypress open` to start the gui.

**Comment by brian-mann** [comment]
Guessing this is likely something with `modifyObstructiveCode`. Set this to false in `cypress.json` and it'll work. This is explained in detail in the changelog. 

https://docs.cypress.io/guides/references/changelog.html

We'll investigate and if that's the cause - we'll look at improving the searches so they don't interfere with valid code.

**Comment by brian-mann** [comment]
Discovered root cause and have a PR open to fix this.

**Comment by brian-mann** [comment]
Released in [`2.0.2`](https://docs.cypress.io/guides/references/changelog.html#2-0-2).
